xcode_common.lua

```diff
@@ -79,25 +79,6 @@
 		return false
 	end
 --
--- Return the displayed name for a build configuration, taking into account the
--- configuration and platform, i.e. "Debug 32-bit Universal".
---
--- @param cfg
---    The configuration being identified.
--- @returns
---    A build configuration name.
---
-
-	function xcode.getconfigname(cfg)
-		local name = cfg.buildcfg
-		if #cfg.project.solution.platforms > 1 then
```

Comment:
You can't have multiple platforms under the XCode hood, so there are no need to have this code.
In reality it will return you something like **Debug Native 64-bit**, however in XCode you'll get only this one, and there will be no way to switch to **Debug Native 32-bit**.
.. or I don't know the way to switch?!